ext/fiddle/fiddle.c

```diff
@@ -58,18 +58,16 @@ rb_fiddle_free(VALUE self, VALUE addr)
 /*
  * call-seq: Fiddle.dlunwrap(addr)
  *
- * Returns the hexadecimal representation of a memory pointer address +addr+
+ * Returns the Ruby object stored at the memory address +addr+
  *
  * Example:
  *
- *   lib = Fiddle.dlopen('/lib64/libc-2.15.so')
- *   => #<Fiddle::Handle:0x00000001342460>
- *
- *   lib['strcpy'].to_s(16)
- *   => "7f59de6dd240"
- *
- *   Fiddle.dlunwrap(Fiddle.dlwrap(lib['strcpy'].to_s(16)))
- *   => "7f59de6dd240"
+ *    >> x = Object.new
+ *    => #<Object:0x0000000107c7d870>
+ *    >> Fiddle.dlwrap(x)
+ *    => 4425504880
+ *    >> Fiddle.dlunwrap(_)
+ *    => #<Object:0x0000000107c7d870>
```

Comment:
I believe we prefer examples syntax that is correct Ruby to be copy-pasted in some file and tried (also RDoc's syntax highlight wouldn't work otherwise, so I'd suggest...
```suggestion
 *    x = Object.new
 *    # => #<Object:0x0000000107c7d870>
 *    addr = Fiddle.dlwrap(x)
 *    # => 4425504880
 *    Fiddle.dlunwrap(addr)
 *    # => #<Object:0x0000000107c7d870>
```